### PR TITLE
Improve question wording

### LINKED ISSUE #140: User guide style: Are the questions appropriate
The questions in each section of the style guide feel (to me) as if they are written for a more technical user.  If we the guide is aimed at such users, then it may be appropriate, but if we want to get more lower ability users using it, then it may be that the questions need re-wording.  Some examples are:
How do I describe inputs to a command?  -->  How do I pass arguments to commands?

How do I describe outputs from a command? -->  Where does my output go, and how do I keep it?

### PR DESCRIPTION
**Pull request by tobyhodges**
Fixes #140

@longr Please take a look at these and let me know what you think.

### CODE REVIEW

File: _episodes/02-1st-example.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "First Example"
 teaching: 5
 exercises: 0
 questions:
-- "How do I wrap a simple command line tool?"
+- "How do I write a CWL wrapper for a simple command line tool?"
```
Comment (mr-c) [inline _episodes/02-1st-example.md]:
```suggestion
- "How do I write a CWL description of a simple command line tool?"
```

File: _episodes/08-arguments.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ teaching: 10
 exercises: 0
 questions:
 - "How do I specify arguments that don't require input values?"
-- "How do I refer to runtime parameters?"
+- "How do I refer to parameters that are only defined at runtime?"
```
Comment (mr-c) [inline _episodes/08-arguments.md]: I think the original question is about [`$(runtime)` ](https://www.commonwl.org/v1.1/CommandLineTool.html#Runtime_environment)

File: _episodes/14-runtime.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ exercises: 0
 questions:
 - "How do I create required input files from input parameters at runtime?"
 - "How do I invoke a script rather than just a simple command line?"
-- "How do I make inputs available to my script?"
+- "How do I pass arguments to my script?"
```
Comment (mr-c) [inline _episodes/14-runtime.md]:
```suggestion
- "Besides `inputBinding`, how else can I pass arguments to the tool?"
```

File: _episodes/16-file-formats.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "File Formats"
 teaching: 10
 exercises: 0
 questions:
-- "How can I allow type-checking of input and output files?"
+- "How do I check that input and output files are formatted correctly?"
```
Comment (mr-c) [inline _episodes/16-file-formats.md]:
```suggestion
- "How can I mark the required file format for input files?"
- "How can I mark the produced file format of output files?"
```

File: _episodes/08-arguments.md (hunk 2 above)
Comment (tobyhodges) [reply _episodes/08-arguments.md]: Sure. Is the question not still valid in that case?

Would you prefer 

> How do I make use of parameters that are only defined at runtime?

?

**Pull request merged**